active_encode.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "aws-sdk-elastictranscoder"
   spec.add_development_dependency "aws-sdk-mediaconvert"
   spec.add_development_dependency "aws-sdk-s3"
-  spec.add_development_dependency "bixby", '~> 4.0'
+  spec.add_development_dependency "bixby", '~> 5.0', '>= 5.0.2'
```

Comment:
This upgrade is necessary to resolve issues with a new rubopcop release that earlier versions of bixby (4.x and 5.x) could not handle.